src/python/WMCore/WMSpec/WMTask.py

```diff
@@ -926,10 +927,19 @@ def setSubscriptionInformation(self, custodialSites = None, nonCustodialSites =
             outputDataset = entry["outputDataset"]
             outputModule = entry["outputModule"]
 
-            primDs = outputDataset.split('/')[1]
-            tier = outputDataset.split('/')[3]
+            dsSplit = outputDataset.split('/')
+
+            primDs = dsSplit[1]
+
+            procDsSplit = dsSplit[2].split('-')
+            skim = ( len(procDsSplit) == 4 )
```

Comment:
> 4 is the number of '-' separated fields in the processed datasets name.

Dahhh, of course! I overlooked it.

The two flags are still not clear to me... how do you interact with them btw, don't you need to provide this parameter from the spec down to the workload and wmtask?

One more comment, pylint won't like this style:
`skim = ( len(procDsSplit) == 4 )`
it should be
`skim = len(procDsSplit) == 4`